samples/graph-appcatalog-lifecycle/csharp/README.md

```diff
@@ -0,0 +1,68 @@
+﻿### AppCatalog sample
+
+Bot Framework v4 AppCatalog sample.
+
+This sample shows a AppCatalog bot and demonstrates teamsApp lifecycle in catalog followed by commands given to Bot.
+
+**Scenarios Covered:**
+- List Apps in catalog
+- Publish App to catalog
+- Update App in catalog
+- Delete App from catalog
+
+## Prerequisites
+
+- Microsoft Teams is installed and you have an account
+- [.NET Core SDK](https://dotnet.microsoft.com/download) version 3.1
+- [ngrok](https://ngrok.com/) or equivalent tunnelling solution
+
+1) Clone the repository
+
+    ```bash
+    git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+    ```
+
+  B) Or from Visual Studio
+
+  - Launch Visual Studio
```

Comment:
Could you please add instruction on setting connection settings? You can refere to [existing doc](https://docs.microsoft.com/en-us/microsoftteams/platform/bots/how-to/authentication/add-authentication?tabs=dotnet%2Cdotnet-sample#azure-ad-v2)  however make sure you specify which permissions are requried.

Reply:
Marking this as unresolved. Could you please work on this?